docs/debezium-aws-rds-config-scripts

```diff
@@ -0,0 +1,361 @@
+= A Step-by-Step Guide on Streaming Data from AWS RDS Multi-AZ PostgreSQL DB to a Standlone PostgreSQL DB Using Debezium 0.7.1 Connector
+fatihmehmetkose
+:awestruct-layout: doc
+:linkattrs:
+:icons: font
+
+# debezium-aws-rds-config-scripts
+![m'lady](https://www.paradigmadigital.com/wp-content/uploads/2017/06/debezium-1.png)
+![m'lady](https://cdn-images-1.medium.com/max/367/1*6j17ZDuywkKu7TOa2yvAKg.png)
+
+
+# A Step-by-Step Guide on Streaming Data from AWS RDS Multi-AZ PostgreSQL DB to a Standlone PostgreSQL DB Using Debezium 0.7.1 Connector :
+
+In this guide, I will share my experience on integrating AWS RDS PostgreSQL and Debezium Kafka 
+Connector. So we will be able to stream data from AWS RDS PostgreSQL to an another standlone 
+PostgreSQL DB on AWS EC2 Server.
+
+Before working with Debezium, you need to make a decision on which connector will be "Source Connector" and 
+which one will be "Sink Connector". In our example, "Source Connector" will connect to "AWS RDS Multi-AZ PostgreSQL DB, 
+and "Sink Connector" will connect to Standlone PostgreSQL DB. 
+
+Actually, the configuration below supports two-way connectors as well. Please keep in mind that only a single 
+Debezium Kafka Connector can serve multiple tasks, it means to be able to use the same connector as source and 
+sink  and can work one-to-many, many-to-one, one-to-one ways. 
+
+Note : You can download the necessary debezium AWS RDS config scripts from https://github.com/fatihmehmetkose/debezium-aws-rds-config-scripts
+
+---
+### Prerequisites:
+To be able to understand all the concepts mentioned here properly, you should be familiar with the systems 
```

Comment:
... you should have a basic understanding of the following technologies:

Reply:
Ok, it's better, I will change.